Mergin/validation.py

```diff
@@ -310,6 +320,12 @@ def check_snapping(self):
                 # snapping in Input using QGIS setting is disabled but project has snapping activated
                 self.issues.append(MultipleLayersWarning(Warning.MERGIN_SNAPPING_NOT_ENABLED))
 
+    def check_datum_shift_grids(self):
+        grids = get_datum_shift_grids()
+        for grid in grids.keys():
+            if not os.path.exists(os.path.join(self.qgis_proj_dir, "proj", grid)):
```

Comment:
I found "proj" in two cases, maybe worth defining util function to get this directory for easier readability?